### ISSUE TITLE
Authorizing a provider already used by another Gitpod account shows confusing popup

### ISSUE THREAD

**Issue by jldec** [gitpod-io/gitpod#7001]
### To Reproduce
- Login to Gitpod
- Go to settings, integrations
- Connect to another provider using a login for which you already have a different Gitpod account

### Questions about the popup (below)
- What will happen when you press Continue? 
- Will something get disconnected?
- Will you be logged out from the current account, and logged into the other account?
- What does it mean to select the "Other Account"

![Screenshot 2021-12-01 at 17 21 23](https://user-images.githubusercontent.com/849592/144284425-1ce13bf8-0a0a-4e84-93cf-ba690825676c.png)

### Suggestion
Treat the situation as simple error notification: 

> Error: you are trying to authorize a provider that is already connected with another Gitpod account.
> To connect with this provider account, you'll first need to login to Gitpod and disconnect or delete the other account.

**Comment by amael-heiligtag** [comment]
Same here, I couldn't figure out what each option will do. And trying both showed me that option 1 was doing nothing (back to square 1), and option 2 leads to an error (asking me to authenticate with a 3rd party OAuth provider - which you can't).

**Comment by bhanu951** [comment]
I also encountered this issue.

I have two github accounts one work and other is personal.
I signed up gitpod with my work and personal accounts.

I wanted to test gitpod with drupal.org so I authenticated git.drupalcode.org , but while doing so I used my work github account instead of my personal account all worked fine for testing the gitpod with drupal.org.

I then realised I authenicated drupal.org with my work account instead of my personal account , so I disconnected gitpod integration in my work account and tried to re-authenicate drupal.org with my personal account.

Now it is showing option to authorize with git.drupalcode.org so I authorized it , and displays option to authorize with other account as shown in image when I try to authorize it is redirecting to login page again.


<img width="546" alt="gitpod1" src="https://user-images.githubusercontent.com/8525040/186177635-7da04e61-9390-4572-a9f7-8945abb4f3c8.png">
<img width="546" alt="gitpod2" src="https://user-images.githubusercontent.com/8525040/186177658-b0d37153-0116-4d54-8d59-32cf3cd547c3.png">
